neurodsp/plts/time_series.py

```diff
@@ -47,18 +47,24 @@ def plot_time_series(times, sigs, labels=None, colors=None, ax=None, **kwargs):
 
     ax = check_ax(ax, (15, 3))
 
-    times = repeat(times) if isinstance(times, np.ndarray) else times
-    sigs = [sigs] if isinstance(sigs, np.ndarray) else sigs
```

Comment:
basically the same update as the `times`: if this line considers dimensionality, then the reshaping in 57 isn't needed:
sigs = [sigs] if (isinstance(arr, np.ndarray) and arr.ndim == 1) else sigs